src/api/app/views/event_mailer/added_user_to_group.html.haml

```diff
@@ -0,0 +1 @@
+"You got added to group '#{event['group']}'"
```

Comment:
You should probably at least wrap this into a `<p>`...

Reply:
Done